src/plugins/console/public/services/settings.ts

```diff
@@ -87,13 +89,22 @@ export class Settings {
     return this.storage.get('disable_history', DEFAULT_SETTINGS.historyDisabled);
   }
 
+  setPollInterval(interval: number) {
+    this.storage.set('poll_interval', interval);
```

Comment:
Nit: it would be a great idea to move these identificators ( 'poll_interval', for example) to the constant variables. What do you think about it?